redirects.json

```diff
@@ -0,0 +1,131 @@
+      "match": "octicons/packages/ruby",
+      "destination": "https://github.com/primer/octicons/tree/main/lib/octicons_gem#readme"
+    },
+    {
+      "name": "Redirect /octicons/packages/jekyll",
+      "match": "octicons/packages/jekyll",
+      "destination": "https://github.com/primer/octicons/tree/main/lib/octicons_jekyll#readme"
+    },
+    {
+      "name": "Redirect /octicons/packages/javascript",
+      "match": "octicons/packages/javascript",
+      "destination": "https://github.com/primer/octicons/tree/main/lib/octicons_node#readme"
+    },
+    {
+      "name": "Redirect /octicons/packages/react",
+      "match": "octicons/packages/react",
+      "destination": "https://github.com/primer/octicons/tree/main/lib/octicons_react#readme"
+    },
+    {
+      "name": "Redirect /octicons/packages/styled-system",
+      "match": "octicons/packages/styled-system",
+      "destination": "https://github.com/primer/octicons/tree/main/lib/octicons_styled#readme"
+    },
+    {
+      "name": "Redirect /octicons/guidelines/design",
+      "match": "octicons/guidelines/design",
+      "destination": "/design/foundations/icons/design-guidelines"
+    },
+    {
+      "name": "Redirect /octicons/*",
+      "match": "(octicons)(.*)",
+      "destination": "/design/foundations/icons{R:2}"
+    }
+  ],
+  "rewrites": [
+    {
+      "name": "ViewComponents Preview assets proxy",
+      "match": "^assets/(.*)",
+      "destination": "https://view-components-storybook.eastus.cloudapp.azure.com/assets/{R:1}"
+    },
+    {
+      "name": "ViewComponents Lookbook assets proxy",
+      "match": "^lookbook-assets/(.*)",
+      "destination": "https://view-components-storybook.eastus.cloudapp.azure.com/lookbook-assets/{R:1}"
+    },
+    {
+      "name": "ViewComponents Lookbook proxy",
+      "match": "^view-components/lookbook/(.*)",
+      "destination": "https://view-components-storybook.eastus.cloudapp.azure.com/view-components/lookbook/{R:1}"
+    },
+    {
+      "name": "ViewComponents Rails App proxy",
+      "match": "^view-components/rails-app/(.*)",
+      "destination": "https://view-components-storybook.eastus.cloudapp.azure.com/view-components/rails-app/{R:1}"
+    },
+    {
+      "name": "ViewComponents proxy",
+      "match": "^view-components/(.*)",
+      "destination": "https://primer.github.io/view_components/{R:1}"
+    },
+    {
+      "name": "Design proxy",
+      "match": "^design/(.*)",
+      "destination": "https://primer.github.io/design/{R:1}"
+    },
+    {
+      "name": "Presentations proxy",
+      "match": "^presentations/(.*)",
+      "destination": "https://primer.github.io/presentations/{R:1}"
+    },
+    {
+      "name": "Doctocat",
+      "match": "^doctocat/(.*)",
+      "destination": "https://primer.github.io/doctocat/{R:1}"
+    },
+    {
+      "name": "CLI proxy",
+      "match": "^cli/(.*)",
+      "destination": "https://primer.github.io/cli/{R:1}"
+    },
```

Comment:
@colebemis given the recent content migrations, can I drop this rewrite now? or do we still need primer.style/cli to be active?